src/main/java/kostka/moviecatalog/service/MovieServiceImpl.java

```diff
@@ -60,6 +60,12 @@ public Movie getMovie(final Long movieId) {
     @Override
     public List<Movie> fullTextSearch(final String searchTerm) {
         List<EsMovie> esMovies = movieEsService.fullTextSearch(searchTerm);
-        return esMovies.stream().map(esMovie -> getMovie(esMovie.getId())).collect(Collectors.toList());
+        List<Movie> movies = new ArrayList<>();
+        try { //upravit
+            movies = esMovies.stream().map(esMovie -> getMovie(esMovie.getId())).collect(Collectors.toList());
```

Comment:
Toto by melo jit pouzit v repository a pak zavolat:
`... findByIdIn(Collection<Integer> ids)`

Reply:
mam na to zalozeny special issue #34